test/unittest/rtps/history/CacheChangePoolTests.cpp

```diff
@@ -33,7 +33,7 @@ class CacheChangePoolTests : public TestWithParam<tuple<uint32_t, uint32_t, Memo
         : pool(nullptr)
         , pool_size(10)
         , max_pool_size(0)
-        , memory_policy(MemoryManagementPolicy_t::PREALLOCATED_MEMORY_MODE)
+        , memory_policy(MemoryManagementPolicy_t::PREALLOCATED_WITH_REALLOC_MEMORY_MODE)
```

Comment:
Same comment as the previous one. Not checking default behavior but it is only an initialization value.